polishNotation.py

```diff
@@ -0,0 +1,39 @@
+def polishNotation(expression):
+    s = stack()
+    result = 0
+    for exp in range(0, len(expression)-1):
+        exp_list.append(exp)
+
+    reverse_exp_list = exp_list[::-1]
+    temp = 0
+
+    for item in reverse_exp_list:
+        if item == "+":
+            operand1 = s.pop()
+            operand2 = s.pop()
+            result = operand1 + operand2

```

Comment:
General comment: you need to validate that both |operand1| and |operand2| are numbers. This applies for all branches so you should think about refactoring this code to avoid so much duplication!

One way is to define a mapping between your operators and a function to handle them. I am using lambda expressions here but you can use normal function (def add(x, y): return x + y):

operators = dict(
  [('+', lambda x, y: x + y),
  ('-', lambda x, y: x - y)])

Here is how to test this:
print(operators['+'](3,5))